backend/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM python:3.12
 WORKDIR /backend
 
 # Copy just the requirements into the working directory so it gets cached by itself
-COPY requirements.txt /backend/requirements.txt
+COPY ./requirements.txt ./requirements.txt
```

Comment:
does this mean line 14 (which I wrote) can be simplified too? i.e. is `backend` not needed?

Reply:
Yes, that's right. I've changed this and pushed the change. Starts-up and run successfully.

Docker build shows it copies successfully....
```
 => [backend 2/5] WORKDIR /backend                                                                                                                                                                                         
 => [backend 3/5] COPY ./requirements.txt ./requirements.txt                                                                                                                                                              
 => [backend 4/5] RUN pip install --no-cache-dir --upgrade -r /backend/requirements.txt                                                                                                                                   
 => [backend 5/5] COPY ./src/. ./src 
```